incapsula/resource_siem_connnection_s3_test.go

```diff
@@ -0,0 +1,81 @@
+package incapsula
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"log"
+	"os"
+	"testing"
+)
+
+//for tests without connectivity check put env variable: TESTING_PROFILE=true
+
+const s3SiemConnectionResourceType = "incapsula_siem_connection_s3"
+const s3SiemConnectionResourceName = "test_acc"
+const s3SiemConnectionResource = s3SiemConnectionResourceType + "." + s3SiemConnectionResourceName
+
+const siemConnectionS3StorageTypeValue = "CUSTOMER_S3"
+
+var s3SiemConnectionName = "SIEMCONNECTIONS3" + RandomLetterAndNumberString(10)
+
+func TestAccS3SiemConnection_Basic(t *testing.T) {
+	log.Printf("========================BEGIN TEST========================")
+	log.Printf("[DEBUG]Running test resource_s3_siem_connection.go.TestAccS3SiemConnection_Basic")
+
+	if v := os.Getenv("TESTING_PROFILE"); v != "" {
```

Comment:
why are you skipping the entire test? can't you just skip the connectivity test?

Reply:
Test creates S3 connection if only access_key and secret_key are valid. If not provided check cannot be done and the test fails. How do you propose just to skip the connectivity test?

Reply:
maybe add an attribute to the api call? or separate the apis in the BE?